ui-v2/package.json

```diff
@@ -48,11 +48,11 @@
 		"humanize-duration": "^3.32.1",
 		"lucide-react": "^0.447.0",
 		"openapi-fetch": "^0.12.2",
-		"react": "^18.3.1",
-		"react-day-picker": "^8.10.1",
-		"react-dom": "^18.3.1",
+		"react": "19.0.0",
+		"react-day-picker": "^9.4.4",
```

Comment:
seems like we don't use this library in the codebase atm. But need to check if this breaks anythiong from v8 to v9